features/step_definitions/helper_services.rb

```diff
@@ -708,7 +708,7 @@
   })
   step 'the step should succeed'
   # cluster-capacity as a target pod
-  step "I run oc create over ERB URL: #{ENV['BUSHSLICER_HOME']}/testdata/infrastructure/cluster-capacity/cluster-capacity-pod.yaml"
+  step 'I run oc create over ERB test file: "/infrastructure/cluster-capacity/cluster-capacity-pod.yaml"'
```

Comment:
Perhaps use `#{BushSlicer::HOME}/testdata/infrastructure/cluster-capacity/cluster-capacity-pod.yaml` ?

Reply:
@liangxia, we don't need to do that anymore, within the step, we are calling the step `I obtain test data file`

Reply:
In case we need to update the location of BushSlicer::HOME in the future, we can update them in batch. Or else, we can miss those ones.

Reply:
@liangxia the [`I obtain test data file` step](https://github.com/openshift/verification-tests/blob/master/features/step_definitions/file.rb#L3-L4) is already doing that, so if we were to add what you suggested, it will lead to invalid path

Reply:
@liangxia this PR should help with a cleared error message for debugging https://github.com/openshift/verification-tests/pull/1025

Reply:
To clarify `test file` here means that a test file from a standard set of test files. If we some day decide to store test files in some other repo again, we can change in one place and download the files instead of copying them.